INN/umbrella-sfpublicpress
# display: initial; for all list item children in posts

## Changes

This pull request makes the following changes:

- Adds `display: initial;` for any element that's a child of `li` inside of post content to fix issues where the items were broken onto a new line from the circle icon if the `li` content is wrapped inside of an element such as
```
<li dir="ltr">
   <p> XXXXXXX </p>
</li>
```

Before:
![Screen Shot 2020-07-01 at 4 03 56 PM](https://user-images.githubusercontent.com/18353636/86286462-7b939b80-bbb4-11ea-87c7-447d08d0a00b.png)

After:
![Screen Shot 2020-07-01 at 4 03 46 PM](https://user-images.githubusercontent.com/18353636/86286465-7c2c3200-bbb4-11ea-9bd5-7ca5e65e960c.png)

## Why



For #141, has no effect on #142 unfortunately

## Testing/Questions

Features that this PR affects:

- Items wrapped inside of an `li` element


Questions that need to be answered before merging:


Steps to test this PR:

1. View this post https://sfpp.test/uncategorized/san-francisco-shelter-in-place-order-could-be-extended-next-week/ (post id 4115) and make sure the list looks ok
2. View a post with a normal list and make sure it looks ok
